modules/nw-networkpolicy-object.adoc

```diff
@@ -36,3 +36,10 @@ only select pods in the project that the NetworkPolicy object is defined.
 <3> A selector matching the pods that the policy object allows ingress traffic
 from. The selector will match pods in any project.
 <4> A list of one or more destination ports to accept traffic on.
+
+[NOTE]
+====
+If you log in as a user with the `cluster-admin` role and access the policy
```

Comment:
It's not specific to cluster-admins, so you can remove `as a user with a cluster-admin role`. (The [first task](https://issues.redhat.com/browse/NETOBSERV-27) was indeed adding this feature just for cluster admins, but the two following tasks [here](https://issues.redhat.com/browse/NETOBSERV-31) and [there](https://issues.redhat.com/browse/NETOBSERV-32) generalized the change for all users)

That being said, since there have been some work on Openshift SDN to catch up with OVN-K network policy features and they are now on par, while this note is technically true, I'm not even sure if it's worth mentioning. I don't know, I let you judge :)

Reply:
Thanks, @jotak. If it's not necessary, I'll remove it. I thought it could have gone either way, but your comments tipped the scale. :-)